### PR TITLE
Feature#28/movie description api

### PR DESCRIPTION
**Pull request by q270311**
I connected MovieDetails component with API

### CODE REVIEW

File: src/getDataFromMovieDB.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ export const getMoviePopular = async ({ page }) => {
 };
 export const getGenres = async () => {
     const response = await axios.get(`https://api.themoviedb.org/3/genre/movie/list?api_key=0a7a30a768304b21322288db13dcdb24&language=en-US`);
```
Comment (kepkaklaudia) [inline src/getDataFromMovieDB.js:12]: We can build these URLs.
Many parts are repeated (e.g. baseURL or API key), so we can declare them above and then reuse them in the code.

File: src/features/moviesList/styled.js [hunk 2]
```diff
@@ -11,4 +12,9 @@ export const Wrapper = styled.div`
     gap: 16px;
     margin-top: 12px;
   }
+`;
+
+export const StyledLink = styled(Link)`
```
Comment (kepkaklaudia) [inline src/features/moviesList/styled.js:17]: We will use StyledLink on other subpages. We can declare it in some file above.

**Review by kepkaklaudia — APPROVED** [review]
Great! Good job!

File: src/getDataFromMovieDB.js (hunk 1 above)
Comment (q270311) [reply src/getDataFromMovieDB.js]: good idea :) I changed it in the code

**Review by kepkaklaudia — APPROVED** [review]
Great! Looks better 😊

**Pull request merged**